### PR TITLE
change(docs): Update User Docs

### PR DESCRIPTION
**Pull request by upbqdn**
## Motivation

We recently updated the docs for Zebra users. I noticed these changes introduced further inconsistencies in the docs.

## Solution

I tried to capture each atomic edit in a single commit. I added comments to particular commits where appropriate.

## Review

Anyone can review. If a particular change isn't suitable, I can revert it.

### Reviewer Checklist

  - [ ] Will the PR name make sense to users?
    - [ ] Does it need extra CHANGELOG info? (new features, breaking changes, large changes)
  - [ ] Are the PR labels correct?
  - [ ] Does the code do what the ticket and PR says?
    - [ ] Does it change concurrent code, unsafe code, or consensus rules?
  - [ ] How do you know it works? Does it have tests?

### CODE REVIEW

File: book/src/SUMMARY.md [hunk 1]
```diff
@@ -16,6 +16,8 @@
   - [Mining](user/mining.md)
     - [Testnet Mining with s-nomp](user/mining-testnet-s-nomp.md)
   - [Kibana blockchain explorer](user/elasticsearch.md)
+  - [Improving Perfomance](user/performance.md)
```
Comment (teor2345) [inline book/src/SUMMARY.md]: Nitpick: spelling

(I haven't reviewed the rest of this PR, please don't block on my review!)

```suggestion
  - [Improving Performance](user/performance.md)
```

File: book/src/user/performance.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+## Improving Performance
+
+Zebra usually syncs in around three days on Mainnet and one afternoon on
```
Comment (mpguerra) [inline book/src/user/performance.md]:
```suggestion
Zebra usually syncs in around three days on Mainnet and half a day on
```

Comment (mpguerra) [reply book/src/user/performance.md]: "afternoon" seems oddly specific and vague at the same 😆

File: book/src/user/performance.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+## Improving Performance
```
Comment (mpguerra) [inline book/src/user/performance.md]: Some of these tips seem like troubleshooting tips. I wonder if we want to merge this section and the "Troubleshooting" section or maybe link to it from Troubleshooting.

File: book/src/user/troubleshooting.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  by default.
+
+## Network Issues
+
+- Some of Zebra's tests download Zcash blocks, so they might be unreliable
+  depending on your network connection. You can set `ZEBRA_SKIP_NETWORK_TESTS=1`
+  to skip the network tests.
+- Zebra may be unreliable on Testnet, and under less-than-perfect network
+  conditions. See our [future
+  work](https://github.com/ZcashFoundation/zebra#future-work) for details.
```
Comment (mpguerra) [inline book/src/user/troubleshooting.md]:
```suggestion
  conditions.
```

Comment (mpguerra) [reply book/src/user/troubleshooting.md]: The "Future Work" section no longer mentions work in these areas. We might also want to consider removing the caveat altogether.

**Review by mpguerra — CHANGES_REQUESTED** [review]
Looks good, thank you! Just a few minor suggestions.

File: book/src/user/performance.md (hunk 3 above)
Comment (upbqdn) [reply book/src/user/performance.md]: I was wavering over the same thing as well. I'll merge the contents of the two sections, remove "Improving Performance" and keep only "Troubleshooting".

My rationale for having those two was that in the future we might be adding more content there, and having it split would prevent the section from being too long.

File: book/src/SUMMARY.md (hunk 1 above)
Comment (upbqdn) [reply book/src/SUMMARY.md]: I'm actually going to put this section into "Troubleshooting", so I'm not going to commit the suggestion.

File: book/src/user/troubleshooting.md (hunk 4 above)
Comment (upbqdn) [reply book/src/user/troubleshooting.md]: I'm going to remove the whole list item.

File: book/src/user/performance.md (hunk 3 above)
Comment (upbqdn) [reply book/src/user/performance.md]: Done in [85a41e2](https://github.com/ZcashFoundation/zebra/pull/6733/commits/85a41e27291c4cc7001a0b8f7706a1b42365075e).

File: book/src/user/troubleshooting.md (hunk 4 above)
Comment (upbqdn) [reply book/src/user/troubleshooting.md]: Done in [291a0b9](https://github.com/ZcashFoundation/zebra/pull/6733/commits/291a0b9fe76aa513b2b73f0e0dcfd0f5af2c561d).

**Review by mpguerra — APPROVED** [review]
Thank you!!

**Pull request merged**